indigo_drivers/aux_ppb/indigo_aux_ppb.c

```diff
@@ -130,7 +137,14 @@ static indigo_result aux_attach(indigo_device *device) {
 		if (AUX_POWER_OUTLET_PROPERTY == NULL)
 			return INDIGO_FAILED;
 		indigo_init_switch_item(AUX_POWER_OUTLET_1_ITEM, AUX_POWER_OUTLET_1_ITEM_NAME, "Power outlets", true);
-		indigo_init_switch_item(AUX_POWER_OUTLET_2_ITEM, AUX_POWER_OUTLET_2_ITEM_NAME, "DSLR outlet", true);
+		indigo_init_switch_item(AUX_POWER_OUTLET_2_ITEM, AUX_POWER_OUTLET_2_ITEM_NAME, "DSLR outlet", true); // In PPBA this can be 0 or 1, but can ALSO be set to variable power 3, 5, 8, 9 or 12 [Volts].
+
+		// PPBA ONLY
+		AUX_POWER_OUTLET_STATE_PROPERTY = indigo_init_light_property(NULL, device->name, AUX_POWER_OUTLET_STATE_PROPERTY_NAME, AUX_GROUP, "Power outlets state", INDIGO_OK_STATE, 4);
```

Comment:
I saw someting more: you initialize AUX_POWER_OUTLET_STATE_PROPERTY with  4 items, but you initialize only 1 item. Am I missing something?

Reply:
Ah, THAT's what that number is for... I needed to read the spec more closely.